charts/incubator/openhab/values.yaml

```diff
@@ -0,0 +1,69 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: openhab/openhab
+  # -- image tag
+  tag: 3.1.0
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See more environment variables in the [openhab documentation](https://openhab.org/docs).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  # TZ: UTC
+  # EXTRA_JAVA_OPTS: -Duser.timezone=UTC
+  # USER_ID: 1000
+  # GROUP_ID: 1000
+  # OPENHAB_HTTP_PORT: 8080
+  # OPENHAB_HTTPS_PORT: 8443
+  # CRYPTO_POLICY: unlimited
+
+# -- Configures service settings for the chart.
+# @default -- See values.yaml
+service:
+  main:
+    ports:
+      http:
+        enabled: true
+        port: 8080
+      https:
+        enabled: true
+        port: 8443
+
+ingress:
+  # -- Enable and configure ingress settings for the chart under this key.
+  # @default -- See values.yaml
+  main:
+    enabled: false
+
+# -- Configure persistence settings for the chart under this key.
```

Comment:
I would put a remark here indicating that the user should select one of the two approaches.